alerts.md

```diff
@@ -222,24 +246,54 @@ Cluster health alerts occur when the cluster health score changes by a specific
 * Out of Memory Pods
 * Failed Jobs
 
-This alert only uses Slack (email coming soon), so it requires the `globalSlackWebhookUrl` field. 
+This alert only uses Slack (email coming soon), so it requires the `globalSlackWebhookUrl` field, or setting the `slackWebhookUrl` field for the alert.
 
 Example Helm values.yaml:
 
 ```
-        # Health Score Alert 
-        - type: health              # Alerts when health score changes by a threshold
-          window: 10m
-          threshold: 5              # Send Alert if health scores changes by 5 or more
+# Health Score Alert 
+- type: health              # Alerts when health score changes by a threshold
+  window: 10m
+  threshold: 5              # Send Alert if health scores changes by 5 or more
 ```
 
-## Alerts Scheduler
+----
+
+## Configuring Alerts in the Kubecost UI
+
+### Cluster Health and Diagnostic Alerts
+
+Cluster Health Alerts and Diagnostic Alerts work differently from other alert types. While other alerts monitor cost data for cost or efficiency anomalies, Health and Diagnostics montior the health of Kubecost itself, as well as the health of the cluster running Kubecost. The UI treats these alert types as "on" or "off", managing a single instance of each, and allowing the settings of this single instance to be adjusted.
+
+![Kubecost Health Alerts](https://raw.githubusercontent.com/kubecost/docs/main/images/health_and_diagnostics_alert_controls.png)
```

Comment:
None of these images are loading for me in preview...

<img width="783" alt="image" src="https://user-images.githubusercontent.com/298359/170307562-250a68ae-8a4e-4c4f-8afc-5eed9ffa57e8.png">

Reply:
That's because they're linked to where the images will exist when we merge.